package.json

```diff
@@ -7,25 +7,31 @@
   "sideEffects": false,
   "type": "module",
   "bin": {
-    "giget": "./dist/cli.mjs"
+    "giget": "./bin/index.mjs"
   },
   "exports": {
     ".": {
-      "import": "./dist/index.mjs",
-      "types": "./dist/index.d.ts",
-      "require": "./dist/index.cjs"
+      "import": {
+        "types": "./dist/index.d.mts",
+        "default": "./dist/index.mjs"
+      },
+      "require": {
+        "types": "./dist/index.d.cts",
+        "default": "./dist/index.cjs"
+      }
     }
   },
-  "main": "./dist/index.cjs",
-  "module": "./dist/index.mjs",
   "types": "./dist/index.d.ts",
+  "main": "./dist/index.cjs",
   "files": [
+    "bin",
     "dist"
   ],
   "scripts": {
     "build": "unbuild",
+    "build:stub": "unbuild --stub",
     "dev": "vitest dev",
-    "giget": "jiti ./src/cli.ts",
+    "start": "node ./bin/index.mjs",
```

Comment:
I like to keep it as is so that don't need stub step. You can move `runMain` to main entry to keep it as before.